tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/es/UserTaskZeebeRecordProcessorElasticSearch.java

```diff
@@ -108,16 +107,8 @@ private UpdateRequest getVariableQuery(final TaskVariableEntity variable)
     try {
       LOGGER.debug("Variable instance for list view: id {}", variable.getId());
       final Map<String, Object> updateFields = new HashMap<>();
-      updateFields.put(
-          TaskVariableTemplate.VALUE,
-          "null".equals(variable.getValue())
-              ? "null"
-              : objectMapper.writeValueAsString(Json.createValue(variable.getValue())));
-      updateFields.put(
-          TaskVariableTemplate.FULL_VALUE,
-          "null".equals(variable.getFullValue())
-              ? "null"
-              : objectMapper.writeValueAsString(Json.createValue(variable.getFullValue())));
+      updateFields.put(TaskVariableTemplate.VALUE, variable.getValue());
```

Comment:
Did you test the situation if the`variable`comes null? 
Is this not gonna throw a null pointer exception?